src/db/entity/record.ts

```diff
@@ -1,11 +1,59 @@
-import { Column, Entity, PrimaryColumn } from 'typeorm';
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import { EmployeeCredentialDto } from 'record/dto/employee.credential.dto';
+import { GovernmentCredentialDto } from 'record/dto/government.credential.dto';
+
+
+  // TODO: add nullable: false where appropriate
 
 @Entity()
 export class Record {
 
-  @PrimaryColumn({ type: 'varchar', length: 100 })
-  name: string;
+  @PrimaryGeneratedColumn()
+  public id: number;
+
+  @Column({ type: 'jsonb' })
+  public entityData: EmployeeCredentialDto | GovernmentCredentialDto;
+
+  @Column({ type: 'varchar', length: 100 })
+  public connection_id: string;
```

Comment:
do we want all of the columns to be 100 chars max or do we want to get more specific for each field?